includes/html/pages/device/port.inc.php

```diff
@@ -74,9 +76,14 @@
 
 $menu_options['graphs'] = 'Graphs';
 $menu_options['realtime'] = 'Real time';
-// FIXME CONDITIONAL
-$menu_options['arp'] = 'ARP Table';
-$menu_options['fdb'] = 'FDB Table';
+
+if (ipv4Mac::where('port_id', $port->port_id)->exists()) {
```

Comment:
Just use the relationships:
`$port->macs()->exists()`
and 
`$port->fdbEntries()->exists()`